projects/batfish/src/main/antlr4/org/batfish/grammar/cisco/CiscoParser.g4

```diff
@@ -1334,6 +1338,7 @@ s_dial_peer
 
 s_domain
```

Comment:
Needs an `enterS_domain` function in `CiscoControlPlaneExtractor` setting `_no` to `ctx.NO() != null`. The `exitS_domain` function should set `_no` to `false`.
Anything `enter.*` or `exit.*`functions for parser rules under `s_domain` need to check value of `_no` before performing action.